Makefile

```diff
@@ -69,6 +70,25 @@ endif
 		-e rolling_update="${ROLLING_UPDATE}" \
 		deploy.yml
 
+attach: check-deploy-env
+	$(eval LIMIT=tag_role_aenode:&tag_env_$(DEPLOY_ENV))
+ifneq ($(DEPLOY_REGION),)
+	$(eval LIMIT=$(LIMIT):&region_$(DEPLOY_REGION))
+endif
+		cd ansible && ansible-playbook \
+		--check \
```

Comment:
check?!

Reply:
it was for testing purposes. remocing now